shared/cfg/base.cfg

```diff
@@ -195,6 +195,12 @@ mem_chk_re_str = ([0-9]+)
 # this domain, hugepage = "yes" is pre-requisite
 # hp_nosharepages = "no"
 
+# Set hugepages on specific host numa node
+# target_nodes = 0
+
+# Number of hugepages to be set on host node0
+# target_num_node0 = 1024
```

Comment:
Explanation looks good to me,

it is better to mention with multiple nodes,
```
# target_nodes = 0,1,2
# target_num_node0 = 1024
# target_num_node1 = 1024
# target_num_node2 = 1024
```

Reply:
@balamuruhans Thanks, updated as you suggested.